tests/api_tests/nodes/test_views.py

```diff
@@ -951,6 +951,103 @@ def test_node_children_list_does_not_include_unauthorized_projects(self):
         assert_equal(len(res.json['data']), 1)
 
 
+class TestNodeChildCreate(ApiTestCase):
+
+    def setUp(self):
+        super(TestNodeChildCreate, self).setUp()
+        self.user = UserFactory.build()
+        self.user.set_password('justapoorboy')
+        self.user.save()
+        self.basic_auth = (self.user.username, 'justapoorboy')
+
+        self.project = ProjectFactory(creator=self.user, is_publc=True)
+
+        self.url = '/{}nodes/{}/children/'.format(API_BASE, self.project._id)
+
+        self.title = 'Cool Project'
+        self.description = 'A Properly Cool Project'
+        self.category = 'data'
+
+        self.user_two = UserFactory.build()
+        self.user_two.set_password('justapoorboy')
+        self.user_two.save()
+        self.basic_auth_two = (self.user_two.username, 'justapoorboy')
+
+        self.public_child = {'title': self.title,
+                             'description': self.description,
+                             'category': self.category,
+                             'public': True}
+        self.private_child = {'title': self.title,
+                              'description': self.description,
+                              'category': self.category,
+                              'public': False}
+
+    def test_creates_public_child_logged_out(self):
+        res = self.app.post_json(self.url, self.public_child, expect_errors=True)
+        # This is 403 instead of 401 because basic authentication is only for unit tests and, in order to keep from
+        # presenting a basic authentication dialog box in the front end. We may change this as we understand CAS
+        # a little better
+        assert_equal(res.status_code, 403)
+
+    def test_creates_public_child_logged_in(self):
```

Comment:
`logged_in_contributor` to match the later test (or `logged_in_owner` or whatever is most appropriate)